keyboards/wheatfield/fanoe_hotswap/keymaps/default/keymap.c

```diff
@@ -0,0 +1,55 @@
+#include QMK_KEYBOARD_H
+#include <print.h>
```

Comment:
```suggestion
```
This can be removed along with the `process_record_user` function definition.

Reply:
OK fixed